packages/venia-ui/lib/components/AddressBookPage/__tests__/__snapshots__/addressBookPage.spec.js.snap

```diff
@@ -69,6 +69,9 @@ exports[`renders correctly when there are existing addresses 1`] = `
       </span>
     </button>
   </div>
+  <AddEditDialog
+    numAddresses={3}
```

Comment:
I honestly don't know why `numAddresses` is the only prop that shows up in the snapshot 🤔

Reply:
React filters out `undefined`, but that's still odd given how many props should be truthy at the point of this snap. I'll play around with this on a local, might actually be exposing a bug; or might be catching an early render cycle. Did @jimbo fix something like this recently by waiting for a full render? `await act(...)` I think.

Reply:
`numAddresses` was needed in order to determine whether or not we should disable the "make this my default" checkbox.

Since we decided not to do that on the front end, this prop has been removed in the end.

Still weird but I didn't dig into it further 😜